assets/svg/icon-bookmark.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="22" height="20" xmlns="http://www.w3.org/2000/svg">
-    <path d="M11.543 18.48a.734.734 0 0 1-.542.247.734.734 0 0 1-.542-.248L2.494 9.811A5.176 5.176 0 0 1 6.17 1c1.719 0 3.32.85 4.284 2.273l.039.059a.592.592 0 0 0 .5.278.577.577 0 0 0 .504-.27A5.164 5.164 0 0 1 15.828 1 5.176 5.176 0 0 1 21 6.17c0 1.36-.521 2.644-1.471 3.615-.005.003-7.986 8.694-7.986 8.694z" stroke="#ADADAD" fill="none" stroke-linecap="round" stroke-linejoin="round"/>
+    <path d="M11.543 18.48a.734.734 0 0 1-.542.247.734.734 0 0 1-.542-.248L2.494 9.811A5.176 5.176 0 0 1 6.17 1c1.719 0 3.32.85 4.284 2.273l.039.059a.592.592 0 0 0 .5.278.577.577 0 0 0 .504-.27A5.164 5.164 0 0 1 15.828 1 5.176 5.176 0 0 1 21 6.17c0 1.36-.521 2.644-1.471 3.615-.005.003-7.986 8.694-7.986 8.694z" stroke="#999999" fill="none" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Would be great if we could have this file optimized as well. This can be done by running `npm run optimize:svg`.

Reply:
✅